assets/input/basic.css

```diff
@@ -88,6 +87,25 @@ textarea[data-valid="false"]:focus {
   @apply text-grey-dark;
 }
 
+input.input_left {
+  @apply rounded-l-4 rounded-r-none;
+}
+
+input.input_right,
+div.input_right {
+  @apply rounded-r-4 rounded-l-none -ml-1;
+}
+
+.input_left:hover,
+.input_right:hover {
+  @apply z-1 transition duration-200;
+}
+
+.input_left:focus,
+.input_right:focus {
+  @apply z-1;
+}
```

Comment:
all of these look like they could be done with pure tailwind classes. can you try that please? as css modules are loaded with the js chunk, we try to use it only when necesssary. most of the values here look that they're likely to be reused somewhere else, so it should have little impact on the global css size

Reply:
as always when changing z-indexes, please give everything that might be overlapping these elements an explicit test run. for example, open a modal on top of it, see if tooltips are never overlapping, check the dropdown menu in the header, etc. (maybe instruct QA to test this as well)

Reply:
will try to use tailwind classes. But just as a sidenote regarding testing. The CSS here is a 1:1 copy from dealerhub and listings. It was duplicated there.
But I'll test again after I applied the tailwind classes

Reply:
The problem here is that rounder-r-none/l-none is overwritten by the global styling of our input component. It did not work to apply the classes in the component itself

Regarding testing, the CSS is a copy from dealerhub and listings (it was duplicated there). But I'll test it again 👍

Reply:
I see. on class-level, the specificity is defined by the order of occurrence in the resulting stylesheet. These selectors might work around exactly that. We should still try reducing the amount of custom CSS as much as possible. I'm open to a compromise

Please make sure after your change, there is no duplicated CSS anymore :)

Reply:
done - removed everything except `rounded-l/r-none` from the global css